docs/API/V1/contents.md

```diff
@@ -23,35 +28,33 @@ The return is a serialized JSON object with information about the named file.
 ## Response status
 
 `401`   **UNAUTHORIZED** \
-The client did not provide an authentication token and there is no public dataset with the resource ID `<dataset>`.
+The client did not provide an authentication token and there is no public
+dataset with the resource ID `<dataset>`.
 
 `403`   **FORBIDDEN** \
-The named `<dataset>` is not public and the authenticated user lacks authorization to read it.
+The named `<dataset>` is not public and the authenticated user lacks
+authorization to read it.
 
 `404`   **NOT FOUND** \
-Either the `<dataset>` or the relative `<path>` within the dataset does not exist
+Either the `<dataset>` or the relative `<path>` within the dataset does not
+exist.
 
 ## Response body
 
-This API returns an `application/json` response body consisting of a JSON object which describes the target `<path>`, in terms of lists of "file" and "directory" JSON objects (as described below).
+This API returns an `application/json` response body consisting of a JSON
+object describing the file or directory at a target `<path>` within a dataset
+tarball.
 
-When the target is a directory, the response object contains a list of subdirectories and a list of files; if the target is a file it returns the file object for that path.
+When the `<path>` refers to a directory, the response object is described in
+[Directory object](#directory-object); when the `<path>` refers to a file, the
+response object is described in [File object](#file-object).
 
-### Directory
+### Directory object
 
-When the `<path>` refers to a directory within the dataset representation, Pbench returns a JSON object containing a list of subdirectory objects (`directories`) and a list of file objects (`files`).
-
-#### Directory object
-
-The directory object gives the name of the directory, and a URI that can be used with a subsequent `GET` operation to get directory data for that nested path.
-
-#### File object
-
-The file object gives the name of the file and a URI that can be used with a subsequent `GET` operation to return the raw byte stream of that file.
-
-Note that symlinks within the dataset representation will result in a URI returning the linked file's byte stream or the linked directory's [directory object](#directory-object).
-
-#### Example
+When the `<path>` refers to a directory within the dataset representation,
+Pbench returns a JSON object with two list fields:
+* `directories` is a list of [subdirectory objects](#subirectory-object), and
+* `files` is a list of [file objects](#file-object)
 
 ```json
 {
```

Comment:
The objects in the `"directories"` list here (at lines 62 and 67) are missing their `"type"` members.